bpf/lib/bpf_linux_binprm.h

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* Copyright Authors of Tetragon */
+
```

Comment:
bit of a nit, but why is this in ./bpf/lib can we put it with the other types ./bpf/process/types/bpf_linux_binprm.h?